<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget {
    background-color: rgb(98, 162, 222); /* Set the background color to a shade of blue */
    color: #ffffff; /* Set the text color to white */
    font-family: Arial, sans-serif; /* Set the font family */
}

/* Apply styles to the individual tabs */
QTabBar::tab {
    background-color:rgb(79, 157, 231); /* Set the background color for each tab */
    color: #ffffff; /* Set the text color for each tab */
    padding: 8px 16px; /* Add padding to the tabs for better spacing */
}

/* Apply styles to the selected tab */
QTabBar::tab:selected {
    background-color: rgb(48, 115, 203); /* Set a different color for the selected tab */
}

/* Apply styles to the content area of the tabs */
QTabWidget::pane {
    border: 1px solid rgb(31, 74, 131); /* Add a border around the content area */
    background-color: rgb(48, 115, 203); /* Set the background color for the content area */
    padding: 10px; /* Add padding to the content area for better spacing */
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Welcome</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 50px; 
font-weight: bold;
}


</string>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 2px 2px; 
font-size: 16px;
font-weight: bold;
}

</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:16px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;Hello Students! Below are some guidelines you are suggested to revise everyday.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>561</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}


</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:16px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Tahoma','sans-serif'; font-size:16px; font-weight:400;&quot;&gt;‣&lt;/span&gt;&lt;span style=&quot; font-size:16px; font-weight:400;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:16px;&quot;&gt;Note: Your Paper Will Be Cancelled If you Minimze your screen.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Tahoma','sans-serif'; font-size:16px; font-weight:400;&quot;&gt;‣&lt;/span&gt;&lt;span style=&quot; font-size:16px; font-weight:400;&quot;&gt; Adhere to high ethical standards, respecting confidentiality and privacy.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Tahoma','sans-serif'; font-size:16px; font-weight:400;&quot;&gt;‣&lt;/span&gt;&lt;span style=&quot; font-size:16px; font-weight:400;&quot;&gt; Maintain accurate and timely submission of questions, ensuring to answer each ascpect asked.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Tahoma','sans-serif'; font-size:16px; font-weight:400;&quot;&gt;‣&lt;/span&gt;&lt;span style=&quot; font-size:16px; font-weight:400;&quot;&gt; Familiarize yourself with and adhere to UI and Flow of the exam system, report any violations or concerns.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Tahoma','sans-serif'; font-size:16px; font-weight:400;&quot;&gt;‣&lt;/span&gt;&lt;span style=&quot; font-size:16px; font-weight:400;&quot;&gt; Prioritize your own physical and mental well-being.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>433</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
cursor: pointer; 
}
QPushButton::hover
{
    background-color:rgb(219, 235, 255);

}</string>
      </property>
      <property name="text">
       <string>Proceed</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Generate Questions</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_37">
         <property name="styleSheet">
          <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 14px; 
font-weight: bold;
}


</string>
         </property>
         <property name="text">
          <string>Subject Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="subject_lineEdit_2">
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
background-color:rgb(255, 255, 255)
}
QLineEdit:hover {
    background-color:rgb(219, 235, 255); 
}</string>
         </property>
         <property name="placeholderText">
          <string>Subject Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 50px; 
font-weight: bold;
}

</string>
      </property>
      <property name="text">
       <string>Questions Generator</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>430</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
cursor: pointer; 
}
QPushButton::hover
{
    background-color:rgb(219, 235, 255);

}</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>441</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_47">
         <property name="styleSheet">
          <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 14px; 
font-weight: bold;
}
</string>
         </property>
         <property name="text">
          <string>Generated Questions are</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_4">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    background-color: rgb(255, 255, 255); 
    border: 2px solid rgb(29, 78, 139); 
    border-radius: 10px; 
    padding: 2px; 
    font-size: 14 px; 
    color: rgb(0, 0, 0); 
}

QTextEdit:hover {
    background-color:rgb(219, 235, 255);
}

</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_38">
         <property name="styleSheet">
          <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 14px; 
font-weight: bold;
}
</string>
         </property>
         <property name="text">
          <string>Difficulty Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="diff_level_2">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: rgb(255, 255, 255); 
    border: 2px solid rgb(29, 78, 139); 
    border-radius: 5px; 
    padding: 5 px; 
    font-size: 14 px; 
    color: rgb(0, 0, 0); 
}

QComboBox:hover {
    background-color:rgb(219, 235, 255); 
}

</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Easy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hard</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Exam</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>400</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QPushButton:hover 
{
background-color: rgb(219, 235, 255); 
}
QPushButton:pressed 
{
background-color: rgb(45, 108, 190); 
color: rgb(255, 255, 255);
box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
}
</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>400</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QPushButton:hover 
{
background-color: rgb(219, 235, 255); 
}
QPushButton:pressed 
{
background-color: rgb(45, 108, 190); 
color: rgb(255, 255, 255);
box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
}
</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QPushButton:hover 
{
background-color: rgb(219, 235, 255); 
}
QPushButton:pressed 
{
background-color: rgb(45, 108, 190); 
color: rgb(255, 255, 255);
box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
}
</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>260</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 40px; 
font-weight: bold;
}

</string>
           </property>
           <property name="text">
            <string>Question</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_5">
           <property name="styleSheet">
            <string notr="true">QTextEdit {
    background-color: rgb(255, 255, 255); 
    border: 2px solid rgb(29, 78, 139); 
    border-radius: 10px; 
    padding: 2px; 
    font-size: 14 px; 
    color: rgb(0, 0, 0); 
}

QTextEdit:hover {
    background-color:rgb(219, 235, 255);
}

</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>-1</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 25px; 
}

</string>
           </property>
           <property name="text">
            <string>Remaining Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_2">
           <property name="styleSheet">
            <string notr="true">QLCDNumber
{
 background-color: rgb(255, 255, 255); 
}
QLCDNumber::hover
{
    background-color:rgb(219, 235, 255);

}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>461</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 40px; 
font-weight: bold;
}

</string>
         </property>
         <property name="text">
          <string>Your Answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_6">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    background-color: rgb(255, 255, 255); 
    border: 2px solid rgb(29, 78, 139); 
    border-radius: 10px; 
    padding: 2px; 
    font-size: 14 px; 
    color: rgb(0, 0, 0); 
}

QTextEdit:hover {
    background-color:rgb(219, 235, 255);
}

</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>481</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget
{
background-color: rgb(219, 235, 255); 
font-color: rgb(21, 50, 88); 
border: 2px solid rgb(48, 115, 203);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QTableWidget:pressed 
{
background-color: rgb(45, 108, 190); 
font-color: rgb(255, 255, 255);
}
</string>
      </property>
      <column>
       <property name="text">
        <string>Subject</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Marks</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Obtained Marks</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Difficulty Level</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>561</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: rgb(255, 255, 255);
font-size: 50px; 
font-weight: bold;
}

</string>
      </property>
      <property name="text">
       <string>Previous Test Results</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QPushButton:hover 
{
background-color: rgb(219, 235, 255); 
}
QPushButton:pressed 
{
background-color: rgb(45, 108, 190); 
color: rgb(255, 255, 255);
box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
}
</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
background-color: rgb(255, 255, 255); 
color: rgb(26, 63, 111); 
border: 2px solid rgb(29, 78, 139);
border-radius: 5px; 
padding: 10px 10px; 
font-size: 16px;
font-weight: bold;
}
QPushButton:hover 
{
background-color: rgb(219, 235, 255); 
}
QPushButton:pressed 
{
background-color: rgb(45, 108, 190); 
color: rgb(255, 255, 255);
box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
}
</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>1136</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: rgb(30, 72, 127);
    font-size: 40px; 
    font-weight: bold;
    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
}

QLabel:hover {
    color: rgb(48, 115, 203);
    text-decoration: underline; 
    cursor: pointer; 
}

QLabel {
    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3); =-
}
</string>
    </property>
    <property name="text">
     <string>Online Examination System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
